packages/verification-vue-component/src/components/Verification/item/cards/SigningCard.vue

```diff
@@ -82,19 +82,37 @@
                         </template>
                     </DataPanel>
                 </div>
+                <div v-if="retractEvents.length > 0 && retractEvents[0].on_blockchain" class="section retraction-hash">
+                    <div class="label">{{ _$t('verification.result.meta.retractionTransaction') }}</div>
+
+                    <DataPanel :key="`expert-${retractEvents[0].ref}`">
+                        <template #header>
+                            <a :href="`https://${net}/tx/${retractEvents[0].on_blockchain.tx_hash}`"
+                               target="_blank">{{ retractEvents[0].on_blockchain.tx_hash }}</a>
+                        </template>
+                    </DataPanel>
+                </div>
             </div>
         </template>
         <template v-else #body>
             <ResultDetail verification-mode="signing"
                           :document-revoked="revokeEvents.length > 0"
                           :document-revocation-in-progress="revocationInProgress"
                           :document-revocation-date="revocationDate"
+                          :document-retracted="retractEvents.length > 0"
+                          :document-retraction-in-progress="retractionInProgress"
+                          :document-retraction-date="retractionDate"
                           :signatures-in-progress="signaturesInProgress"
                           :has-unverified-signer="hasUnverifiedSigner"
                           :has-verified-signer="hasVerifiedSigner"
                           :signer-count="signEvents.length"/>
 
             <div class="verification-info">
+                <div v-if="retractEvents.length > 0" class="section retraction-note">
+                    <span class="note-label" v-html="_$t('verification.result.meta.retractionNote')"/>
+                    <span class="note-value" v-html="retractionNote" />
```

Comment:
I wouldn't inject the note with `v-html`, this allows code injection. I think at the moment we don't allow HTML inside the note, so better to just normally inject it. It's also not needed for the label.
Also I don't really agree with the usage of `span` here. I would probably use a `div` for the label and a `p` or `div` for the value. Is the `note-` prefix needed? Couldn't you just call it `label` and `value`?

Reply:
Makes sense, changed it